src/Chainweb/Pact/Backend/RelationalCheckpointer.hs

```diff
@@ -335,11 +335,13 @@ doLookupSuccessful curHeight hashes = do
     callDb "doLookupSuccessful" $ \db -> do
       let
         hss = V.toList hashes
-        params = Utf8 $ intercalate "," (map (const "?") hss)
-        qtext = "SELECT blockheight, hash, txhash FROM \
-                \TransactionIndex INNER JOIN BlockHistory \
-                \USING (blockheight) WHERE txhash IN (" <> params <> ")"
-                <> " AND blockheight <= ?;"
+        params = intercalate "," (map (const "?") hss)
+        qtext = Utf8 $ intercalate " "
```

Comment:
`intercalate " "` is just `unwords`

Reply:
Oh, it's Bytestring:

> Module ‘Data.ByteString’ does not export ‘unwords’.

Reply:
Ah I see, makes sense seeing as it's binary data.